### DISCUSSION TITLE
Cache Issue

### DISCUSSION THREAD

**Discussion by yodhacube** [microweber/microweber#1098] [General]
Hi @peter-mw 

We import data content that we set as product (the flow is not from admin). The database has been updated but it seems that on the content detail / product detail page the add to cart button has not appeared.

The add to cart button only appears after we delete the cache from storage/cache.

Is there a better way to refresh the cache other than manually deleting the folder?

**Comment by peter-mw** [comment]
hi, you can clear the cache  by  going on url `http://example.com/api/clearcache`  or using the `clearcache()` PHP function

**Reply by yodhacube** [reply]
Thanks @peter-mw !
